client/lib/signup/step-actions.js

```diff
@@ -157,6 +157,10 @@ export function createSiteWithCart(
 		newSiteParams.blog_name = importingFromUrl;
 		newSiteParams.find_available_url = true;
 		newSiteParams.public = 1;
+	} else if ( flowName === 'onboarding' ) {
+		newSiteParams.blog_name = get( user.get(), 'username', '' ) + siteType;
```

Comment:
I think we should remove `siteType` from this.

Reply:
Done.